vivbin

```diff
@@ -63,7 +63,12 @@ def main():
             verbose = True
 
         elif opt == "-O":
-            vw.config.parseConfigOption( optarg )
+            try:
+                vw.config.parseConfigOption( optarg )
+            except:
+                print vw.config.reprConfigPaths(True)
```

Comment:
Also, might be worth trimming this down to printing something like "invalid value for option foo" so users know which one was problematic.